src/options/options.css

```
@@ -20,51 +19,64 @@ body.mobile .mobile-incompatible {
   opacity: 0.4;
 }
 
-ul {
-  margin: 0px;
-  padding: 0px;
+form {
+  margin: 16px 32px;
 }
-li {
-  list-style-type: none;
-  margin-top: 10px;
-  padding: 0px;
+
+section {
+  margin-bottom: 32px;
 }
-/* in a fieldset e.g. we use a condensed version */
-.condensed-list {
-  margin-left: 0px;
+
+h1 {
+  margin: 32px 0 0 0;
```

So why not just...?
```suggestion
  margin-top: 32px;
```
(untested)

(same for many others)